lib/elixir/pages/typespecs.md

```diff
@@ -8,6 +8,7 @@ Elixir comes with a notation for declaring types and specifications. Elixir is a
 Type specifications (sometimes referred to as *typespecs*) are defined in different contexts using the following attributes:
 
   * `@type`
+  * `@typedoc`
```

Comment:
I’m not sure it belongs here, this one accepts docs string, all the other ones the typespec "syntax"